src/DurableTask.Netherite.AzureFunctions/DurableTask.Netherite.AzureFunctions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
I need to update the TFM here to successfully build, as DTFx.Core only supports .NET 6. However, I'm wondering if we should also consider updating the TFM for Netherite. @sebastianburckhardt . If not, we might need to consider some other ways here.

Reply:
+1.
Put differently, it makes sense to have the TFM of the `Netherite.AzureFunctions` project match the "WebJobs extension".
And have the `DurableTask.Netherite` TFM match `DurableTask`

Reply:
Right now [DurableTask.Netherite ](https://github.com/microsoft/durabletask-netherite/blob/main/src/DurableTask.Netherite/DurableTask.Netherite.csproj#L4)supports _netstandard2.1;netstandard2.0;netcoreapp3_. while [DurableTask.Core](https://github.com/Azure/durabletask/blob/main/src/DurableTask.Core/DurableTask.Core.csproj#L5) supports only netstandard2.0. I didn't update that yet in this PR, I am a little confused if I should update that too.

Reply:
> I am a little confused if I should update that too.

I am not actually sure what the consequences of removing it are, i.e. who would be affected and how / why.

Reply:
But maybe this is a good moment to remove unnecessary stuff since it a major version release.

Reply:
I think we can leave it as is if you agree, given that targeting netstandard2.1;netstandard2.0;netcoreapp3.x is stricter and has a narrower scope compared to netstandard2.0. @sebastianburckhardt

Reply:
I would be happy to try to replace `netstandard2.1;netstandard2.0;netcoreapp3.x` with just ` netstandard2.0` ... I don't think we need all of those. 

But I am uncomfortable doing that without some testing.

Reply:
I can handle the testing. What specific tests do you think would be valuable, aside from the usual smoke and performance tests?

Reply:
So my suggestion for now: change this to only `netstandard2.0`

And we will run a few tests (which we should do anyway) before the next release to make sure everything still works.

Reply:
I have some additional load tests that are a bit more work (they are not fully automated) so I do only run them right before making a release. 

Are you planning to make a release soon?

Reply:
Thanks. I am starting this mainly to catch up with the next functions bundles release. But if this takes time, I think it's ok to catch up the next bundles release. :)

Reply:
Upgrade the TFM :)